README.md

```diff
@@ -44,6 +45,32 @@ You can view a demo of a few ways to use these helpers [here](http://locusenergy
 
 ## Usage
 
+### `{{d3-graph graph}}`
+
+`d3-graph` is the only component in this addon. It's used to provide root level selection where your graph will be rendered.
```

Comment:
What do you think about rewording this sentence - to something like:
"It's used to provide root level selection to render discrete D3 elements, such as SVG `<svg>` and groups `<g>`."